### PR TITLE
Optimizing monitor scan costs on Flex Licensing

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Guidance on optimizing monitor scan costs on Flex Licensing

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-477

### CODE REVIEW

File: docs/alerts/monitors/monitor-faq.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ description: Frequently asked questions about Sumo Logic monitors.
...
 
+## How can I optimize scan costs for monitors when using Flex Pricing?
+
+To optimize scan costs for monitors under [Flex Pricing](/docs/manage/partitions/flex), consider the following factors:
+
+- **Data scanned by the query**. This is the primary driver of cost and is incurred every time the monitor is evaluated. To reduce costs, optimize your query using [default scope](/docs/manage/partitions/flex/faq/#how-can-i-optimize-my-query-using-default-scope) to include only necessary partitions and minimize the amount of data scanned.
+- **Time range of the monitor query**. For static monitors, adjust the detection window under [Trigger Type](/docs/alerts/monitors/create-monitor/#step-1-set-trigger-conditions) (for example, `"Alert when result is _____ within <detection window> minutes"`) to use a shorter time range, which reduces the amount of data scanned. For outlier monitors, reduce the **datapoints** parameter under **Trigger Type** to lower the scanned bytes.
+- **Frequency of monitor evaluation**. Currently, the frequency at which a monitor is evaluated is not configurable. However, it's essential to be mindful that more frequent evaluations will lead to higher scan costs.
```
Comment (banant) [inline docs/alerts/monitors/monitor-faq.md:15]: Let's delete this line for now.

Comment (kimsauce) [inline docs/alerts/monitors/monitor-faq.md:15]:
```suggestion
```

File: docs/alerts/monitors/monitor-faq.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ description: Frequently asked questions about Sumo Logic monitors.
...
+
+By carefully configuring these elements, you can optimize monitor performance while keeping scan costs under control.
```
Comment (banant) [inline docs/alerts/monitors/monitor-faq.md:16]: By carefully configuring these elements, you can balance scan costs with monitoring requirements.

File: docs/alerts/monitors/monitor-faq.md (hunk 1 above)
Comment (kimsauce) [reply docs/alerts/monitors/monitor-faq.md]: Done!

**Review by banant — COMMENTED** [review]
I added two comments

File: docs/alerts/monitors/monitor-faq.md [hunk 3]
```diff
@@ -6,6 +6,15 @@ description: Frequently asked questions about Sumo Logic monitors.
...
 import AlertsTimeslice from '../../reuse/alerts-timeslice.md';
 
+## How can I optimize scan costs for monitors when using Flex Pricing?
+
+To optimize scan costs for monitors under [Flex Pricing](/docs/manage/partitions/flex), consider the following factors:
+
+- **Data scanned by the query**. This is the primary driver of cost and is incurred every time the monitor is evaluated. To reduce costs, optimize your query using [default scope](/docs/manage/partitions/flex/faq/#how-can-i-optimize-my-query-using-default-scope) to include only necessary partitions and minimize the amount of data scanned.
+- **Time range of the monitor query**. For static monitors, adjust the detection window under [Trigger Type](/docs/alerts/monitors/create-monitor/#step-1-set-trigger-conditions) (for example, `"Alert when result is _____ within <detection window> minutes"`) to use a shorter time range, which reduces the amount of data scanned. For outlier monitors, reduce the **datapoints** parameter under **Trigger Type** to lower the scanned bytes.
+
+By carefully configuring these elements, you can optimize monitor performance while keeping scan costs under control.
```
Comment (kimsauce) [inline docs/alerts/monitors/monitor-faq.md:16]:
```suggestion
By carefully configuring these elements, you can balance scan costs with monitoring requirements.
```

File: docs/alerts/monitors/monitor-faq.md (hunk 2 above)
Comment (kimsauce) [reply docs/alerts/monitors/monitor-faq.md]: perfect! fixed.

**Comment by kimsauce** [comment]
> I added two comments

Thank you so much for the quick response! May I publish, @banant ?

**Pull request merged**